.github/workflows/inspektor-gadget.yml

```diff
@@ -296,6 +296,42 @@ jobs:
           rm -rf /tmp/.buildx-cache
           mv /tmp/.buildx-cache-new /tmp/.buildx-cache
 
+  scan-gadget-container-images:
+    name: Scan gadget container images
+    # level: 1
+    needs: build-gadget-container-images
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        type: [ default, core ]
+        os: [ linux ]
+        platform: [ amd64 ]
```

Comment:
Out of curiosity: why not arm64 too?

Reply:
Nothing particular. I assumed doing it just for `amd64` would be enough. What do you think?

Reply:
Yes, I think only checking a specific architecture should be good enough. AFAIU it'd very very weird to have a CVE only for one architecture.